memstore/kafka_redolog_manager.go

```diff
@@ -0,0 +1,168 @@
+package memstore
+
+import (
+	"encoding/json"
+	"github.com/Shopify/sarama"
+	"github.com/uber/aresdb/utils"
+	"math"
+	"sync"
+)
+
+const maxBatchesPerFile = 5000
```

Comment:
should this be configurable?

Reply:
for now, this number should be fixed otherwise saved progress (consist of fileID and offset) is not useable after this number is changed.